app/views/finders/show.html.erb

```diff
@@ -58,6 +58,10 @@
   <% end %>
 </header>
 
+<% if finder.summary %>
+  <div class="keyline"><!-- --></div>
```

Comment:
Would an `<hr>` tag be more appropriate semantically? "a paragraph-level thematic break", which is what the commit suggests this is for.